driver/ssh/config.go

```diff
@@ -17,7 +18,7 @@ type Config struct {
 func (conf *Config) EncodeWithDSN(base string) (string, error) {
 	parsed, err := url.Parse(base)
 	if err != nil {
-		return "", fmt.Errorf("url parsing the base url: %s, %s", base, err.Error())
+		return "", fmt.Errorf("url parsing the base url: %s", err.Error())
```

Comment:
Maybe we can make these error messages a bit more friendly :smile: 

```suggestion
		return "", fmt.Errorf("DSN base isn't a parseable URL: %v", err)
```